dali/operators/util/reshape.cc

```diff
@@ -28,34 +29,75 @@ DALI_SCHEMA(Reshape)
   .AllowSequences()
   .SupportVolumetric()
   .AddOptionalArg<int>("shape", "The desired shape of the output. Number of elements in "
```

Comment:
It would be nice to provide the information, that Reshape can also take optional input with shapes. Now there is no such information.